packages/teams-js/test/public/location.spec.ts

```diff
@@ -293,8 +293,8 @@ describe('location', () => {
     });
 
     it('Frameless - getLocation should throw error when not supported in the runtime config', () => {
-      utils.setRuntimeConfig({ apiVersion: 1, supports: {} });
```

Comment:
How did these work before?

Reply:
Long story, but essentially initializeWithContext used to behave very oddly because the majority of people were passing no HostClientType into it. That then tried to respond to the initialize message with an undefined HostClientType in the response which caused assorted "weirdness" because a supposedly required parameter in the response (HostClientType) was missing. Runtime was also in that response but after the messed up HostClientType value, which (I think) was causing this `setRuntimeConfig` call to not get wiped out by processing the initialize request.

Now that `initializeWithContext` *always* sets HostClientType, you need to set runtime config after that call is finished.